application/app/src/main/java/com/example/buber/Model/User.java

```diff
@@ -2,9 +2,10 @@
 
 
 public abstract class User {
-    public static enum TYPE {
-        Drivers,                //Firebase clollection names
-        Riders
+    public enum TYPE {
+        //Firebase clollection names
+        Drivers,
```

Comment:
Not sure but I think it should be Rider or Driver, and from my understanding, I think the only reason we have this enum is to tell firebase what the type of user it is so I don't think encapsulation should be questioned?
Since rider and driver extend user we set the enum type when the rider or driver class is created